### PR TITLE
[5.2] Admin Module Latest

### PR DESCRIPTION
**Pull request by brianteeman**
### Summary of Changes
The admin module Articles - Latest displays a list of recently added content. This is really useful on the dashboards. (especially when used together with the author filter)

The list can be sorted by created or modified date but it will only ever display the created date.

This PR changes the date column to display created or modified depending on the sort param.

### Testing Instructions
Create some articles with varying created and modified dates
Change the ordering option in the module

### Actual result BEFORE applying this Pull Request
The list of articles are sorted based on the selected date but the displayed date is always the created date

### Expected result AFTER applying this Pull Request
The list of articles are sorted based on the selected date but the displayed date is the same as the sort

image

image

### No new param
I guess this could have been done by adding a new param for the displayed date but 
1. it is more logical that you want to display the date you are sorting by
2. because the modified date is now available you can create a template override to display the date how you want - even to display both dates

### Link to documentations
Please select:
- [ ] Documentation link for docs.joomla.org: <link>
- [ ] No documentation changes for docs.joomla.org needed

- [ ] Pull Request link for manual.joomla.org: <link>
- [ ] No documentation changes for manual.joomla.org needed

### CODE REVIEW

File: administrator/modules/mod_latest/tmpl/default.php [hunk 1]
```diff
@@ -25,7 +25,10 @@
                 <th scope="col" class="w-20"><?php echo Text::_('JSTAGE'); ?></th>
             <?php endif; ?>
             <th scope="col" class="w-20"><?php echo Text::_('JAUTHOR'); ?></th>
-            <th scope="col" class="w-20"><?php echo Text::_('JDATE'); ?></th>
+            <th scope="col" class="w-20">
+                <?php echo ($params->get('ordering', 'c_dsc') == 'm_dsc') ? Text::_('MOD_LATEST_HEADING_DATE_MODIFIED') : Text::_('MOD_LATEST_HEADING_DATE_CREATED'); ?>
+            </th>
+
         </tr>
```
Comment (Quy) [inline administrator/modules/mod_latest/tmpl/default.php]:
```suggestion
            </th>
        </tr>
```

**Comment by Quy** [comment]
I have tested this item :white_check_mark: successfully on ce05fd61eac900eef71ec6bdb3c3c52122c60e96<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/43216">issues.joomla.org/tracker/joomla-cms/43216</a>.</sub>

**Comment by carlitorweb** [comment]
I have tested this item :white_check_mark: successfully on ce05fd61eac900eef71ec6bdb3c3c52122c60e96<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/43216">issues.joomla.org/tracker/joomla-cms/43216</a>.</sub>

**Comment by Quy** [comment]
RTC<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/43216">issues.joomla.org/tracker/joomla-cms/43216</a>.</sub>

**Comment by pe7er** [comment]
Thanks @brianteeman !

**Comment by brianteeman** [comment]
thx

**Pull request merged**